src/core/transaction.spec.ts

```diff
@@ -729,29 +710,28 @@ describe("test transaction", async () => {
 
     it("should compute bytes to verify transaction signature (signed by hash)", async () => {
         let transaction = new Transaction({
-            sender: wallets.alice.address,
-            receiver: wallets.bob.address,
+            sender: alice.address,
+            receiver: bob.address,
             gasLimit: 50000n,
             chainID: "D",
             nonce: 7n,
         });
 
         transactionComputer.applyOptionsForHashSigning(transaction);
 
-        transaction.signature = await wallets.alice.signer.sign(transactionComputer.computeHashForSigning(transaction));
+        transaction.signature = await alice.sign(transactionComputer.computeHashForSigning(transaction));
 
-        const userVerifier = new UserVerifier(new UserPublicKey(wallets.alice.address.getPublicKey()));
+        const userVerifier = new UserVerifier(alice.publicKey);
```

Comment:
Can use `account.verify()`?

Now I see, that we don't have separate `verify_transaction` and `verify_message`:
 - https://github.com/multiversx/mx-sdk-specs/blob/main/accounts/account.md

How about having so?

Reply:
I will add this but in the next pr if that's ok. I don't want to add functionality on this pr as is enough big as it is